### ISSUE TITLE
Installer doesn't work

### ISSUE THREAD

**Issue by benthurley82** [WorksOfBarry/ILEditor#82]
Hi @WorksOfLiam,

I just tried installing on my Win 10 laptop but the installer doesn't seem to work. I get a popup saying "Verifying application requirements. This may take a few moments." but a split second later the popup vanishes and nothing else happens.

I'm sorry it's not much to go on. Let me know if there's any debug output I can get for you to help.

Cheers
Ben

**Comment by worksofliam** [comment]
Hey Ben

I had a quick look into this today but should be able to give a better update tomorrow.

Probably something .NET related and is hopefully a simple fix.

Thanks!
Liam

**Comment by worksofliam** [comment]
Hi @benthurley82 

After some Googling there is a few options available.

1. Running as admin (right click-> Run as Administrator)
2. Disabling or uninstalling Dell System Detect: http://www.dell.com/support/article/uk/en/ukbsdt1/sln117738/dell-supportassist-formerly-dell-system-detect-overview-and-common-questions?lang=en (towards the bottom)

Please let me know how you get on.

Thanks!

**Comment by benthurley82** [comment]
Ok it seems to work now. I'm not 100% sure what fixed it. Yesterday I tried downloading the .net core library version linked from the ILEditor homepage but it didn't fix anything. However, I've since rebooted so that could have done the trick. I did already have .net core installed but different versions. 
Just now I tried running as admin and the app started but whether that was because I was running as admin or due to the .net core dependency I don't know.
